client_app/src/components/shelter/NavBarShelterDashboard.js

```diff
@@ -37,6 +37,11 @@ function CustomNavBarShelter({ auth }) {
                   active={location.pathname === "/request-for-help"}>
                   Set Shifts
                 </Nav.Link>
+                <Nav.Link
+                  href="/upcoming-requests"
+                  active={location.pathname === "/upcoming-requests"}>
+                  Upcoming Shifts
+                </Nav.Link>
                 <Nav.Link href="/cancel-shifts" active={location.pathname === "/cancel-shifts"}>
```

Comment:
please remove the cancel shifts from the navigation bar

Reply:
I didn't add this Nav link, will that component be used later in a different sprint? I can delete it right now though, I just wanted to confirm.

Reply:
Instead of Upcoming Requests, it was mentioned Cancel shifts. So we need to change cancel shifts in the nav bar to upcoming request since you already added upcoming shifts Please remove the cancel shifts from the navigation bar.